server.js

```diff
@@ -0,0 +1,24 @@
+console.log('chuck\'s computer is loading express')
+
+const express = require('express');
+const app = express();
+//const bodyParser = require('body-parser)').urlencode({estended: true});
+const PORT = process.env.PORT || 3000;
+app.use(express.static('/portfolio'));
+
+//was unsure how to enable bodyParser since I do not have a form or articles, I have read that it may be required to parse json data. Is that correct?
```

Comment:
We shouldn't need `bodyParser` for anything on your portfolio, but if you do decide to use it you're probably better off wiring it up as [middleware for all requests](https://github.com/DeltaVCode-cr-301-2017/09-sql-joins-relations/blob/17c656384ac6e1c5e865c9afb3c30446692f6ab2/starter-code/server.js#L16-L17).